File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-05/plagiarized/L4/05/method.ts
import { Scanner } from 'readline-sync';

class Method {
  // method reverse
  public static beautyReverse(num: number): void {
    while (num !== 0) {
      const r = num % 10;
      console.log(r);
      num = Math.floor(num / 10);
    }
    console.log();
  }

  static reverse(n: number): void {
    Method.beautyReverse(n);
  }

  // main program
  public static main(args: string[]): void {
    console.log('Enter an integer: ');
    const input = new Scanner();
    const n = input.nextInt();
    // call method
    Method.reverse(n);
  }
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-05/plagiarized/L4/05/method.py
import sys

def beauty_reverse(num):
    while num != 0:
        r = num % 10
        print(r, end='')
        num = num // 10
    print()

def reverse(n):
    beauty_reverse(n)

if __name__ == "__main__":
    print("Enter an integer: ", end="")
    n = int(input())
    reverse(n)

